<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>运动员动作智能分析系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/prefix1/m_icon.png</normaloff>:/prefix1/m_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1261</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="3,2" rowminimumheight="180,300">
     <item row="0" column="0">
      <widget class="QGroupBox" name="pInfo">
       <property name="title">
        <string>运动员信息选择</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>591</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QGraphicsView" name="pImage"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,3">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="topMargin">
            <number>20</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="select_info" stretch="1,1,1">
             <property name="spacing">
              <number>15</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>Microsoft YaHei UI</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>运动员选择</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>Microsoft YaHei UI</family>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>分析角度选择</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="select_button" stretch="1,1,1">
             <property name="spacing">
              <number>15</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QComboBox" name="comboBox_2"/>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox"/>
             </item>
             <item>
              <widget class="QPushButton" name="query">
               <property name="text">
                <string>查询</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="select_date" stretch="1,1,1">
             <property name="spacing">
              <number>15</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QDateEdit" name="dateEdit"/>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEdit_2"/>
             </item>
             <item>
              <widget class="QPushButton" name="analysis">
               <property name="text">
                <string>分析</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="dbLayout" stretch="1">
       <item>
        <widget class="QTabWidget" name="dbView">
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_1">
          <attribute name="title">
           <string>右侧方向</string>
          </attribute>
          <widget class="QTableWidget" name="tableView">
           <property name="geometry">
            <rect>
             <x>-1</x>
             <y>-1</y>
             <width>751</width>
             <height>421</height>
            </rect>
           </property>
           <property name="cursor" stdset="0">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>左侧方向</string>
          </attribute>
          <widget class="QTableWidget" name="tableView2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>751</width>
             <height>421</height>
            </rect>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>前侧方向</string>
          </attribute>
          <widget class="QTableWidget" name="tableView3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>751</width>
             <height>421</height>
            </rect>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QGroupBox" name="analysis_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>分析数据</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QVideoWidget" name="video_ns" native="true">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>341</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="image_body">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>250</y>
          <width>201</width>
          <height>351</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
